.github/workflows/continuous-deployment.yml

```
@@ -0,0 +1,142 @@
+          token: ${{ secrets.GH_TOKEN }}
+      - run: git fetch --tags --unshallow
+
+      - name: Check commit
+        if: ${{ github.ref != 'refs/heads/develop' && github.ref != 'refs/heads/master' }}
+        uses: oat-sa/conventional-commit-action@v0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
```

actionlint (1.7.10)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)